spec/src/main/asciidoc/release_notes.asciidoc

```diff
@@ -33,6 +33,7 @@ There are no functional changes introduced in this release, except the dependenc
 
 * Negative Test Scenario - @SchemaProperty Precedence Behaviour (link:https://github.com/eclipse/microprofile-open-api/issues/466[466])
 * Use MediaType.APPLICATION_JSON instead of application/json in some TCKs (link:https://github.com/eclipse/microprofile-open-api/pull/471[471])
+* Use entry identity in list immutability check (TCK) (link:https://github.com/eclipse/microprofile-open-api/pull/488[488])
```

Comment:
@MikeEdgar Can you use issue number #453 instead? We try to stick with issue if possible.

Reply:
@Emily-Jiang , updated the link. Thanks